packages/wdio-testingbot-service/src/launcher.ts

```diff
@@ -14,20 +14,24 @@ export default class TestingBotLauncher {
         this.options = options
     }
 
-    async onPrepare (config: WebdriverIO.Config) {
+    async onPrepare (config: WebdriverIO.Config, capabilities: WebDriver.Capabilities) {
         if (!this.options.tbTunnel || !config.user || !config.key) {
             return
         }
 
+        const tbTunnelIdentifier = `TB-tunnel-${Math.random().toString().slice(2)}`
+
         this.tbTunnelOpts = Object.assign({
             apiKey: config.user,
-            apiSecret: config.key
+            apiSecret: config.key,
+            'tunnel-identifier': tbTunnelIdentifier,
         }, this.options.tbTunnelOpts)
 
-        config.protocol = 'http'
-        config.hostname = 'localhost'
-        config.port = 4445
-        config.path = '/wd/hub'
+        if (!capabilities['tb:options']) {
+            capabilities['tb:options'] = {}
+        }
+
+        capabilities['tb:options'].tunnelIdentifier = tbTunnelIdentifier
```

Comment:
This probably will not work if 2 tests are running in parallel, so I'll revert this and create my null PR